README.md

```diff
@@ -18,6 +18,8 @@ Or install it yourself as:
 
```

Comment:
Might be good to add a bit of background in the intro blurb to help orientate people who stumble upon this. Perhaps a little background about (or link to) what the presenter pattern is? Also your motivation for creating the gem and how the opinions evolved?